types/luxon/luxon-tests.ts

```diff
@@ -196,8 +196,8 @@ if (Interval.isInterval(anything)) {
 
 /* Info */
 Info.months();
-// $ExpectError
-Info.months('2-digit', { outputCalendar: 'unknown' });
+// // $ExpectError
+// Info.months('2-digit', { outputCalendar: 'unknown' });
 Info.weekdays('long');
```

Comment:
No, sorry, we should not do this (we should not hide the problems). What are those errors you've mntioned in commit?

Reply:
@peterblazejewicz Agree, I didn't like removing the test but could not find a way to make it pass. It seems like the library itself doesn't throw an error if you run that function, but TypeScript does. The property that's being passed is expected to be a string from a specific set of strings, which 'unknown' doesn't belong to.

If I run it as is, the test says `TypeScript@4.3: Expected an error on this line, but found none.`
If I remove the $ExpectError, it says `Compile error in typescript@4.1 but not in typescript@4.3. Fix with a comment '// Minimum TypeScript Version: 4.3' just under the header.`

I'm not sure where that's supposed to go or if it's actually the right thing to do; this is my first PR to DefinitelyTyped and this issue is unrelated to the change I was submitting.

Reply:
this is because of this PR landing in 4.3: microsoft/TypeScript#42885
the `calendar` is now relaxed `string` type, as per specs. Can you remove those line entirely, pls?
thx!

Reply:
Sure, thanks for helping me out.